java/src/jmri/jmrit/signalling/SignallingSourcePanel.java

```diff
@@ -180,7 +169,10 @@ public void propertyChange(java.beans.PropertyChangeEvent e) {
             boolean newValue = (Boolean) e.getNewValue();
             discoverPairs.setEnabled(newValue);
         }
-        //log.debug("SSP 179 Event: {}", e.getPropertyName());
+        log.debug("SSP 173 Event: {}; Source: {}", e.getPropertyName(), e.toString()); // doesn't get notified, newDestination
+        if (e.getPropertyName().equals("Frame.active")) { // a blunt way of redrawing the Pairs table
```

Comment:
Thanks @rhwood In hindsight it was rather easy to fix being guided to the correct place (chose Default_SML_Manager).
PS: it is possible to open an unlimited number of SML Pairs panes. Now that all these panes refresh as new items become available, I think that's no big deal:
![multiplesmledis](https://cloud.githubusercontent.com/assets/14811327/22798799/a229bb70-ef03-11e6-8d8f-d8011f7156ff.png)